.github/workflows/build-ci-atlas.yml

```diff
@@ -0,0 +1,76 @@
+name: "Atlas CI"
+
+on:
+    push:
+    pull_request:
+
+jobs:
+    build:
+        runs-on: "${{ matrix.os }}"
+
+        name: "PHP ${{ matrix.php }} Laravel ${{ matrix.laravel }} Atlas"
+
+        strategy:
+            matrix:
+                os:
+                    - "ubuntu-latest"
+                mongodb:
+                    - "Atlas"
```

Comment:
I don't see any reference to `matrix.mongodb` in this file. Is this needed for something else? If not, I think we can just remove the `mongodb` matrix axis.